CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+### 2.1.0 (unreleased)
```

Comment:
In the changelogs up until now I've included with each item the relevant PR number. I think it's a nice detail for people who want to look into the details behind a decision. In this text file it would be more cumbersome to do because `#1234` is not automatically converted into a link. The information will be there in the git blame, but that's quite a bit harder to find.

What do you think?

Reply:
I'd go with Mardown links, since those will be properly formatted in both GitHub and Discourse. I didn't originally add them due to laziness, but they are a worthy addition.

Reply:
I love the result but I'm concerned it'll be cumbersome for contributors. Let's see how it works.